helm/app/templates/dashboard-role.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ include "app.fullname" $ }}
+  {{- if hasKey $values "namespace" }}
+  namespace: {{ $values.namespace }}
+  {{- end }}
+  labels:
+    helm.sh/chart: {{ include "app.chart" $ }}
+    app.kubernetes.io/name: {{ include "app.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+rules:
```

Comment:
maybe whole rules should be configurable as well from values file

Reply:
I think as best practice it's kept in the templates as seen in gocd charts:
[gocd-ea-cluster-role](https://github.com/helm/charts/blob/master/stable/gocd/templates/gocd-ea-cluster-role.yaml)